src/rgw/driver/d4n/d4n_policy.cc

```diff
@@ -489,6 +497,8 @@ void LFUDAPolicy::update(const DoutPrefixProvider* dpp, const std::string& key,
   }
 
   weightSum += ((localWeight < 0) ? 0 : localWeight);
+  for (const auto& key : entries_map) 
+    ldpp_dout(dpp, 0) << "entries_map: " << key << dendl;
```

Comment:
so the entries become incorrect at this point and is always correct before, correct?

Reply:
and is there a test which can trigger the failure?